### ISSUE TITLE
Expose all `chief` functionality

### ISSUE THREAD

**Issue by nmushegian** [makerdao/governance-dashboard#67]
I understand the primary purpose of the UI is to guide users toward a particular common / simpler use pattern, but we must provide a way to call all functions. Remember, the GUI can never be the definition of functionality, since people have direct access to contracts.

For example, I'd like to create a slate with more than one candidate. I'd like to be able to modify my proposal votes in the UI, since they're not blocked on the actual contracts.

*At the very least, we need an "Advanced" tab before promoting this over chief.makerdao.com - otherwise this is misleading people about how voting works*

**Comment by nmushegian** [comment]
Low priority...

mariano.conti M 11:02 AM   
@nikolai you can already do that if you have Metamask and go to the Write Contract tab
 nikolai 11:00 AM
Imagine the day when etherscan will inject a web3 and let you write to contracts
